github_test.go

```diff
@@ -147,3 +151,60 @@ func TestGitHubPullRequest_comment(t *testing.T) {
 		t.Log(*c.CommitID)
 	}
 }
+
+func TestGitHubPullRequest_Post_Flash_mock(t *testing.T) {
+	apiCalled := 0
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.URL.String() != "/repos/haya14busa/reviewdog/pulls/2/comments" {
+			t.Errorf("unexpected access: %v %v", r.Method, r.URL)
+			return
+		}
+
+		switch r.Method {
+		case "GET":
+		case "POST":
+			var v github.PullRequestComment
+			if err := json.NewDecoder(r.Body).Decode(&v); err != nil {
+				t.Error(err)
+			}
+			body := *v.Body
+			want := `<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
+[reviewdog] test`
+			if body != want {
+				t.Error("body: got %v, want %v", body, want)
```

Comment:
**[govet]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
possible formatting directive in Error call